client-linuxapp/src/serviceinfo.rs

```diff
@@ -70,6 +75,48 @@ fn create_user(user: &str) -> Result<()> {
     Ok(())
 }
 
+// Returns true if password is encrypted
+// is_password_encrypted functionality is taken from osbuild-composer's crypt.go
```

Comment:
nit: might be nice to include a link to the implementation in crypt.go

Reply:
Resolved, link added